### PR TITLE
flask-restplus: update to 0.10

### PR DESCRIPTION
**Pull request by langston-barrett**
###### Motivation for this change
Tested and working

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/flask-restplus/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  version = "0.10.1";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1qs9c3fzidd0m3r8rhw0zqrlsaqr2561z45xs6kg19l7c2x6g5qj";
+  };
+
+  checkInputs = [ nose blinker tzlocal mock rednose ];
+  propagatedBuildInputs = [ flask six jsonschema pytz aniso8601 flask-restful ];
```
Comment (FRidh) [inline pkgs/development/python-modules/flask-restplus/default.nix]: you added itself to `propagatedBuildInputs`

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/flask-restplus/default.nix (hunk 1 above)
Comment (siddharthist) [inline pkgs/development/python-modules/flask-restplus/default.nix]: Ah, what a ridiculous mistake! Thanks for the catch.

Comment (siddharthist) [inline pkgs/development/python-modules/flask-restplus/default.nix]: Wait, I don't think I did. You may be referring to `flask-restful`? These are different projects:
 * https://github.com/flask-restful/flask-restful
 * https://github.com/noirbizarre/flask-restplus

Comment (FRidh) [inline pkgs/development/python-modules/flask-restplus/default.nix]: AHH, indeed, my bad.

**Pull request merged**